docs/data/material/guides/composition/composition.md

```diff
@@ -22,6 +22,47 @@ WrappedIcon.muiName = Icon.muiName;
 
 {{"demo": "Composition.js"}}
 
+### Forwarding slot props
+
+Use the `mergeSlotProps` utility function to merge custom props with the slot props.
+If the arguments are functions, they are resolved before merging. The result from the first argument will override the second.
```

Comment:
This is a rare example where it's ok to use the future tense "will" because it's an "if... then" scenario. I would also connect these two sentences into one if I'm understanding it correctly.

```suggestion
If the arguments are functions then they'll be resolved before merging, and the result from the first argument will override the second.
```